src/pages/home/platform-carousel/_mobile-platform-carousel.tsx

```diff
@@ -74,28 +75,40 @@ const settings = {
 
 const image_query = graphql`
     query {
-        platforms_deriv_go: file(relativePath: { eq: "home/platforms_deriv_go.png" }) {
+        platforms_deriv_go: file(relativePath: { eq: "home/platforms_deriv_go_m.png" }) {
```

Comment:
having it named `mobile` instead of `m` would be more descriptive for others in the future.